devel/shards/Portfile

```diff
@@ -27,11 +38,15 @@ pre-fetch {
   }
 }
 
+post-extract {
+  file mkdir ${worksrcpath}/lib
+  file copy ${workpath}/${molinillo_dist} ${worksrcpath}/lib/molinillo
+}
```

Comment:
This whitespace doesn't conform to the modeline. Indentations should be at multiples of 4 spaces.